src/SFA.DAS.ApplyService.Data/ApplyRepository.cs

```diff
@@ -121,16 +120,156 @@ public async Task SubmitApplication(Guid applicationId, ApplyData applyData, Gui
             using (var connection = new SqlConnection(_config.SqlConnectionString))
             {
                 await connection.ExecuteAsync(@"UPDATE Apply
-                                                SET  ApplicationStatus = @ApplicationStatus, ApplyData = @applyData, ReviewStatus = @ReviewStatus, UpdatedBy = @submittedBy, UpdatedAt = GETUTCDATE() 
+                                                SET  ApplicationStatus = @ApplicationStatus, ApplyData = @applyData, AssessorReviewStatus = @ReviewStatus, GatewayReviewStatus = @GatewayReviewStatus, UpdatedBy = @submittedBy, UpdatedAt = GETUTCDATE() 
```

Comment:
Looks like FinancialReviewStatus is missing

Reply:
It would be set to the default value defined in the table definition, but it's a good idea to document what the initial state of the application is in code.